allensdk/brain_observatory/ecephys/copy_utility/__main__.py

```diff
@@ -170,4 +168,4 @@ def main(
     )
 
     output = main(**parser.args)
-    write_or_print_outputs(output, parser)
+    parser.output(output)
```

Comment:
can you do `parser.output(indent=2)` ?
this helps it be readable

Reply:
Done!